draft-packages/well/Well.tsx

```diff
@@ -18,6 +18,8 @@ export type WellProps = {
   variant?: VariantType
   borderStyle?: BorderStyleType
   noMargin?: boolean
+  straightCorners?: boolean
```

Comment:
There should be some guidance or comments on when to use `straightCorners`. The interface in Avatar is a good baseline: https://github.com/cultureamp/kaizen-design-system/blob/master/draft-packages/avatar/KaizenDraft/Avatar/Avatar.tsx#L26-L45